tests/libnet/cloudinit.go

```diff
@@ -197,3 +197,19 @@ func CreateDefaultCloudInitNetworkData() (string, error) {
 		),
 	)
 }
+
```

Comment:
Regarding the commit message. The static ipv6 config on `CreateDefaultCloudInitNetworkData` is relevant for masquerade binding only. There is no reason for bridge binding to use it (and using it is not better in any way than having dhcp6 enabled).

Reply:
Well, that's not exactly true; while my commit message is clearly wrong, that is not used only for masquerade: it is used also for sr-iov, as you can see in https://github.com/kubevirt/kubevirt/blob/727dec427340b79cd8a76b94f2f7b0393e916c99/tests/vmi_multus_test.go#L1363

That is why I'm adding a new method instead of having a single one - I started with one; when things started to go south on places not related to masquerade, I created a new one.